docs/commerce/checkout/checkout.md

```diff
@@ -0,0 +1,51 @@
+---
+description: The checkout component covers providing shipping and billing addresses, and selecting payment and shipping methods.
+edition: commerce
+---
+
+# Checkout
+
+Checkout is a crucial component of the Commerce offering delivered as part 
+of [[= product_name =]].
+In a course of a multi-step process, it collects necessary transaction data, such 
+as billing and shipping addresses, and payment and shipping information.
+
+From the front-end perspective, it is a reusable component that provides access 
+to the workflow and allows buyers to place an order for cart items.
+
+![Address selection stage](img/checkout.png "Checkout stages")
+
+Depending on the model of commerce presence that you decide upon, the checkout 
+process can range between a straightforward and extremely complicated one. 
+To allow for this variation, the component is highly configurable and extensible:
+
+- Like the editorial workflow, it relies on [Symfony Workflow](../../content_management/workflow/workflow.md) 
+- It exposes [PHP API](checkout_api.md) that allows for workflow manipulation
+- It exposes TWIG functions used for checkout rendering
+
+In a default implementation, users go through a series of steps.
+They first select a billing and shipping address, then select shipping and payment 
+methods, later they review summary and confirm an order, to finally receive an order 
+confirmation.
```

Comment:
Technically, we don't create any order per se, it's part of upcoming feature. It's worth to mention that to avoid confusion I think.

Reply:
I reworded this to "later they review summary and confirm their choices, to finally receive 
a simulated order confirmation".